.drone.yml

```diff
@@ -1,80 +1,132 @@
 ---
 kind: pipeline
 name: linux-amd64
+
 platform:
   os: linux
   arch: amd64
 
 steps:
-- name: ci
-  image: briandowns/rancher-build-base:v0.1.1
+- name: build
```

Comment:
Looks like this and the arm `ci` pipeline are doing the same thing for diff arch's so I'm confused by the naming. This is doing more than just build right?

Reply:
Changed back to `ci`.